src/managers/GuildScheduledEventManager.js

```diff
@@ -72,7 +78,7 @@ class GuildScheduledEventManager extends CachedManager {
         description,
         entity_type: entityType,
         entity_metadata: {
```

Comment:
Just pass in options.entityMetadata instead

Reply:
This will have to be changed in the future in case other properties in snake case are added inside the object, I think we should just put them all in one object instead of having this metadata object which seems to just be an api thing with no real use

Reply:
Sure, let's keep the object as is and manually provide variables but we don't know what may come to the entityMetadata field in the future, and we try to stay as close to the API as possible

Reply:
I know we try to stay close to the API but this wasn’t done with threads simply because it causes unnecessary burden and doesn’t bring any benefits to the users (at least that’s how I see it, if there was a better reason there please let me know) so I don’t see why we can’t do the same here

Reply:
Resolving this because this has been dealt with. A similar conversation is open above for further discussion.